app/tests/utils.py

```diff
@@ -230,3 +233,30 @@ def recurse_callbacks(callbacks, django_capture_on_commit_callbacks):
             callbacks=new_callbacks,
             django_capture_on_commit_callbacks=django_capture_on_commit_callbacks,
         )
+
+
+def create_raw_upload_image_session(
+    *,
+    django_capture_on_commit_callbacks,
+    image_paths: list[str],
+    delete_files: list[str] = None,
+    user=None,
+    linked_task=None,
+) -> tuple[RawImageUploadSession, dict[str, UserUpload]]:
```

Comment:
I made this function more generic and promoted it to an official "utils" function for it to be available across tests. I hope that is alright?

Reply:
Would prefer refactoring to be separate from feature PRs, but it's fine.